packages/blitz-paint/src/render.rs

```diff
@@ -395,6 +400,11 @@ struct ElementCx<'a> {
     devtools: &'a DevtoolSettings,
 }
 
+///TODO: If BoundingBox implements `Shape`, remove this. This exists to side-step it not currently doing that.
```

Comment:
`BoundingBox` is never going to implement `Shape`. So maybe just make this comment describe what the function does.